### PR TITLE
[sample] Sending Teams meeting invites

### PR DESCRIPTION
**Pull request by AlexJerabek**
This sample shows how to schedule a series of Teams meetings based on an Excel table.

### CODE REVIEW

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+localization_priority: Normal
+---
+
+# Office Scripts sample scenario: Schedule interviews in Teams
+
+In this scenario, you're an HR recruiter scheduling interview meetings with candidates in Teams. You manage the interview schedule of candidates in an Excel file. You'll need to send the Teams meeting invite to both the candidate and interviewers. You then need to update the Excel file with the confirmation that Teams meetings have been sent.
+
+The solution has three steps that are combined in a single Power Automate flow.
+
+1. An script extracts data from a table and returns an array of objects as JSON data.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:14]:
```suggestion
1. A script extracts data from a table and returns an array of objects as JSON data.
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+## Sample code: Mark rows as invited
+
+Name this script **Record Sent Invites** for the flow.
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook, invites: InterviewInvite[]) {
+  const table = workbook.getWorksheet("Interviews").getTables()[0];
+
+  // Get the ID and Invite Sent columns from teh table.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:97]:
```suggestion
  // Get the ID and Invite Sent columns from the table.
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  FinishTime: string
+  Message: string
+}
+```
+
+## Sample flow: Run the interview scheduling scripts and send the Teams meetings
+
+1. Create a new **Instant cloud flow**.
+1. Select **Manually trigger a flow** and press **Create**.
+1. Add a **New step** that uses the **Excel Online (Business)** connector and the **Run script** action. Complete the connector with the following values:
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:134]:
```suggestion
1. Add a **New step** that uses the **Excel Online (Business)** connector and the **Run script** action. Complete the connector with the following values.
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    1. **Location**: OneDrive for Business
+    1. **Document Library**: OneDrive
+    1. **File**: hr-interviews.xlsx *(Chosen through the file browser)*
+    1. **Script**: Schedule Interviews
+    :::image type="content" source="../../images/schedule-interviews-1.png" alt-text="The completed Excel Online (Business) connector to get interview data from the workbook in Power Automate":::
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:139]:
```suggestion
    :::image type="content" source="../../images/schedule-interviews-1.png" alt-text="Screenshot of the completed Excel Online (Business) connector to get interview data from the workbook in Power Automate":::
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+1. Add a **New step** that uses the **Create a Teams meeting** action. As you select dynamic content from the Excel connector, an **Apply to each** block with be generated for your flow. Complete the connector with the following values:
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:140]:
```suggestion
1. Add a **New step** that uses the **Create a Teams meeting** action. As you select dynamic content from the Excel connector, an **Apply to each** block will be generated for your flow. Complete the connector with the following values.
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    1. **Calendar id**: Calendar
+    1. **Subject**: Contoso Interview
+    1. **Message**: **Message** (the Excel value)
+    1. **Time zone**: Pacific Standard Time
+    1. **Start time**: **StartTime** (the Excel value)
+    1. **End time**: **FinishTime** (the Excel value)
+    1. **Required attendees**: **CandidateEmail** ; **InterviewerEmail** (the Excel values)
+    :::image type="content" source="../../images/schedule-interviews-2.png" alt-text="The completed Teams connector to schedule meetings in Power Automate":::
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:148]:
```suggestion
    :::image type="content" source="../../images/schedule-interviews-2.png" alt-text="Screenshot of the completed Teams connector to schedule meetings in Power Automate":::
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+1. In the same **Apply to each** block, add another **Excel Online (Business)** connector with the **Run script** action. Use the following values:
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:149]:
```suggestion
1. In the same **Apply to each** block, add another **Excel Online (Business)** connector with the **Run script** action. Use the following values.
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+    1. **Location**: OneDrive for Business
+    1. **Document Library**: OneDrive
+    1. **File**: hr-interviews.xlsx *(Chosen through the file browser)*
+    1. **Script**: Record Sent Invites
+    1. **invites**: **result** (the Excel value)
+    :::image type="content" source="../../images/schedule-interviews-3.png" alt-text="The completed Excel Online (Business) connector to record that invites have been sent in Power Automate":::
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:155]:
```suggestion
    :::image type="content" source="../../images/schedule-interviews-3.png" alt-text="Screenshot of the completed Excel Online (Business) connector to record that invites have been sent in Power Automate":::
```

File: docs/resources/scenarios/schedule-interviews-in-teams.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+1. Save the flow and try it out.
+
+## Training video: Send a Teams meeting from Excel data
+
+[Watch Sudhi Ramamurthy walk through a version this sample on YouTube](https://youtu.be/HyBdx52NOE8). His version uses a more robust script that handles changing columns and obsolete meeting times.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/scenarios/schedule-interviews-in-teams.md:160]:
```suggestion
[Watch Sudhi Ramamurthy walk through a version of this sample on YouTube](https://youtu.be/HyBdx52NOE8). His version uses a more robust script that handles changing columns and obsolete meeting times.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

File: docs/resources/scenarios/schedule-interviews-in-teams.md (hunk 2 above)
Comment (AlexJerabek) [reply docs/resources/scenarios/schedule-interviews-in-teams.md:97]: It really bothers me that the Docs Authoring Pack doesn't flag "teh" as a spelling error...

Comment (ElizabethSamuel-MSFT) [reply docs/resources/scenarios/schedule-interviews-in-teams.md]: It thought it was "meh" which is perfectly fine...lol

**Pull request merged**